text/game/loadingtips.stringtable

```diff
@@ -416,32 +416,32 @@
     </Entry>
     <Entry>
       <ID>88</ID>
-      <DefaultText>Verwende die Tab-Taste, um interaktive Gegenstände in der Umgebung zu markieren.</DefaultText>
+      <DefaultText>Verwende die Tabtaste, um interaktive Gegenstände in der Umgebung anzuzeigen.</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>89</ID>
-      <DefaultText>Wenn du die Försterhütte bei deiner Festung baust, erhältst du Zugang zu zahlreichen Kopfgeldaufträgen. Kopfgeld-Quests sind herausfordernde Kampfbegegungen, die wertvolle Belohnungen einbringen kann, darunter auch einzigartige Gegenstände.</DefaultText>
+      <DefaultText>Wenn du die Försterhütte bei deiner Festung baust, erhältst du Zugang zu zahlreichen Kopfgeldaufträgen. Kopfgeldqests sind herausfordernde Kampfbegegungen, die wertvolle Belohnungen einbringen können, darunter auch einzigartige Gegenstände.</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>90</ID>
-      <DefaultText>NSCs mit goldenen Namen wurden von Backern gestaltet. Interagiere mit ihnen um die Möglichkeit zu haben diese einmaligen Geschichten zu erleben.</DefaultText>
+      <DefaultText>NPCs mit goldener Namensplakette wurden von Backern gestaltet. Interagiere mit ihnen, um die Möglichkeit zu haben, die einzigartige Geschichten jedes einzelnen dieser NPCs zu ergründen.</DefaultText>
```

Comment:
NSC ist auch ein feststehender Begriff spätestens dann wenn du mal was mit
Rollenspielen fern ab vom Computer zu tun hast, denn Nicht-Spieler
Charakter ist ein feststehender Begriff in der Pen & Paper Welt. Die
Abkürzung dafür ist NSC.

Mit freundlichem Gruß

Martin Berkemeier

Web: http://berkemeier-it.de
Mail: martin@berkemeier-it.de

Am 9. April 2015 um 23:46 schrieb LordCrash88 <notifications@github.com>:

> In text/game/loadingtips.stringtable
> <https://github.com/phob/pillarsofeternity-german-patch/pull/87#discussion_r28106034>
> :
>
> >        <FemaleText />
> >      </Entry>
> >      <Entry>
> >        <ID>90</ID>
> > -      <DefaultText>NSCs mit goldenen Namen wurden von Backern gestaltet. Interagiere mit ihnen um die Möglichkeit zu haben diese einmaligen Geschichten zu erleben.</DefaultText>
> > +      <DefaultText>NPCs mit goldener Namensplakette wurden von Backern gestaltet. Interagiere mit ihnen, um die Möglichkeit zu haben, die einzigartige Geschichten jedes einzelnen dieser NPCs zu ergründen.</DefaultText>
>
> Ja, nur existiert IMHO NSC als Begriff überhaupt nicht. In meinen knapp 15
> Jahren Beschäftigung mit RPGs ist mir das nie unter gekommen. Für mich ist
> NPC ein feststehender Begriff, der von jedermann verstanden wird, während
> das imho für NSC nicht im gleißen Maße gilt. Hier geht also
> Verständlichkeit vor Übersetzungswut. ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phob/pillarsofeternity-german-patch/pull/87/files#r28106034>
> .
>